client/components/emojify/docs/example.jsx

```diff
@@ -12,12 +12,12 @@ const EmojifyExample = () => {
 	const textToEmojify = 'This 🙈 will be converted 🙉🙊🙂';
 
 	return (
-		<div className="design-assets__group">
+		<div imgClassName="design-assets__group">
```

Comment:
this isn't a valid attribute for a div. did you mean to place it on the `Emojify` component?